app/pyproject.toml

```diff
@@ -118,3 +116,33 @@ reportUntypedBaseClass = false
 reportUntypedClassDecorator = true
 reportUntypedFunctionDecorator = false
 reportUnusedCallResult = false
+
+[tool.ruff]
+# # Enable Pyflakes `E` and `F` codes by default.
+select = ["F", "E", "W", "I", "N", "YTT", "A", "C4", "RET", "SIM"]
+ignore = [
+    # "A002",
+    "A003",
+    "E402",
+    "E741",
+    "F403",
+    "N802",
+    "N803",
+    "N806",
+    "N815",
+    "RET503",
+    "RET504",
+    "RET505",
+    "SIM102",
+]
+
+# Exclude a variety of commonly ignored directories.
+exclude = [
+    "migrations",
```

Comment:
previously migrations were not excluded, do we want to keep that?

Reply:
Looks like they were: https://github.com/mozilla/experimenter/blob/main/app/.flake8#L2